src/docs/automation-and-resiliency/application-resource-tuning.md

```diff
@@ -0,0 +1,286 @@
+---
+title: Application Resource Tuning  
+
+slug: application-resource-tuning
+
+description: Describes how to tune resource limits and requests in OpenShift.
+
+keywords: Tuning, Resource, Limit, Request, Application OpenShift,
+
+page_purpose: Discusses application tuning, and provides resources and references for product teams.
+
+audience: developer
+
+author: Matt Spencer
+
+content_owner: 
+
+sort_order: 2
+---
+
+# Application Resource Tuning
+
+As touched upon in the [Resiliency Guidelines](/App-Resiliency-Guidelines#resourced), deploying applications with appropriate CPU and Memory [Requests and Limits](https://docs.openshift.com/container-platform/3.11/admin_guide/overcommit.html#requests-and-limits) is critical to ensure:
+
+* Resource availability for your own applications
+* Resource availability for other tenant applications
+
+While [Resource Quotas](./src/docs/automation-and-resiliency/openshift-project-resource-quotas) are quite generous, these quotas must be seen as a tool to allow tenants enough resources to temporarily burst usage for experimentation, rather than an upper limit of consistent use. The platform is not sized to support every tenant fully utilizing their _Resource Quota_.
+
+**Resource Requests**  
+Resource requests are guaranteed and reserved for the pod. _Pod scheduling decisions are made based on the request_ to ensure that a node has enough capacity available to meet the requested value. Inefficient use of requests lead to having to buy more licenses and hardware for the platform.
+
+**Resource Limits**  
+Resource limits set an upper limit of what a pod can burst to if the resources are available on the node.
+
+## Setting Requests and Limits
+
+❗ **If you set a resource limit, you should also set a resource request.** Otherwise the Request will match the limit. For example, a Deployment with _no_ defined CPU request and a defined CPU limit of 1 core will _result in a pod with a request of 1 CPU and a limit of 1 CPU_.
```

Comment:
```suggestion
❗ **If you set a resource limit, you should also set a resource request.** Otherwise the request will match the limit. For example, a deployment with _no_ defined CPU request and a defined CPU limit of one core will _result in a pod with a request of one CPU and a limit of one CPU_.
```